build/edge/run_daemon.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+if [ ! -d /etc/kubeedge/edge/certs ] || [ ! -e /etc/kubeedge/edge/certs/edge.crt ] || [ ! -e /etc/kubeedge/edge/certs/edge.key ]; then
+    mkdir -p /etc/kubeedge/edge/certs
+    mkdir -p /etc/kubeedge/ca
+    docker run --rm \
+    -v /etc/kubeedge/ca:/etc/kubeedge/ca \
+    -v /etc/kubeedge/edge/certs:/etc/kubeedge/certs \
+    kubeedge/certgen:v0.2 genCertAndKey edge
```

Comment:
as we had discussed in other PR , we may not need a special docker image for certificate generation etc. Please see if we can incorporate this via some common script utilities .

Reply:
OK. I correct here to use a script not the image.